npm-shrinkwrap.json

```diff
@@ -585,6 +585,96 @@
+                          "version": "2.0.0",
+                          "from": "https://registry.npmjs.org/ansi-regex/-/ansi-regex-2.0.0.tgz",
+                          "resolved": "https://registry.npmjs.org/ansi-regex/-/ansi-regex-2.0.0.tgz"
+                        }
+                      }
+                    },
+                    "strip-ansi": {
+                      "version": "3.0.1",
+                      "from": "https://registry.npmjs.org/strip-ansi/-/strip-ansi-3.0.1.tgz",
+                      "resolved": "https://registry.npmjs.org/strip-ansi/-/strip-ansi-3.0.1.tgz",
+                      "dependencies": {
+                        "ansi-regex": {
+                          "version": "2.0.0",
+                          "from": "https://registry.npmjs.org/ansi-regex/-/ansi-regex-2.0.0.tgz",
+                          "resolved": "https://registry.npmjs.org/ansi-regex/-/ansi-regex-2.0.0.tgz"
+                        }
+                      }
+                    },
+                    "supports-color": {
+                      "version": "2.0.0",
+                      "from": "https://registry.npmjs.org/supports-color/-/supports-color-2.0.0.tgz",
+                      "resolved": "https://registry.npmjs.org/supports-color/-/supports-color-2.0.0.tgz"
+                    }
+                  }
+                },
+                "dbug": {
+                  "version": "0.4.2",
+                  "from": "https://registry.npmjs.org/dbug/-/dbug-0.4.2.tgz",
+                  "resolved": "https://registry.npmjs.org/dbug/-/dbug-0.4.2.tgz"
+                },
+                "stack-trace": {
+                  "version": "0.0.9",
+                  "from": "https://registry.npmjs.org/stack-trace/-/stack-trace-0.0.9.tgz",
+                  "resolved": "https://registry.npmjs.org/stack-trace/-/stack-trace-0.0.9.tgz"
+                },
+                "strftime": {
+                  "version": "0.9.2",
+                  "from": "https://registry.npmjs.org/strftime/-/strftime-0.9.2.tgz",
+                  "resolved": "https://registry.npmjs.org/strftime/-/strftime-0.9.2.tgz"
+                },
+                "symbol": {
+                  "version": "0.2.1",
+                  "from": "https://registry.npmjs.org/symbol/-/symbol-0.2.1.tgz",
+                  "resolved": "https://registry.npmjs.org/symbol/-/symbol-0.2.1.tgz"
+                },
+                "utcstring": {
+                  "version": "0.1.0",
+                  "from": "https://registry.npmjs.org/utcstring/-/utcstring-0.1.0.tgz",
+                  "resolved": "https://registry.npmjs.org/utcstring/-/utcstring-0.1.0.tgz"
+                }
+              }
+            },
+            "merge": {
+              "version": "1.2.0",
+              "from": "https://registry.npmjs.org/merge/-/merge-1.2.0.tgz",
+              "resolved": "https://registry.npmjs.org/merge/-/merge-1.2.0.tgz"
+            }
+          }
+        },
```

Comment:
The appearance of moziog @2.0.3 here for auth-mailer, and below for i18n-abide, is due to mozlog directly used by auth-server moving to mozlog@2.0.5. There's a bug in 2.0.3 for which auth-mailer and i18n-abide should be updated.